### PR TITLE
Execute the required cds builds in each module

### PR DESCRIPTION
**Pull request by MarcelMerkle**
This ensures that the required artifacts are generated before the mta build packages the respective modules

### CODE REVIEW

**Comment by beckermarc** [comment]
Would it also be an option to run mvn clean install at the beginning of the whole MTAR build once? 
Then defining these dependencies and calling mvn in the srv build should not be required anymore.

**Comment by MarcelMerkle** [comment]
@beckermarc Could you have another look?

File: srv/pom.xml [hunk 1]
```diff
@@ -111,7 +111,7 @@
 						</goals>
 						<configuration>
 							<commands>
-								<command>build/all --clean</command>
+								<command>build --for java-cf --clean</command>
```
Comment (beckermarc) [inline srv/pom.xml:114]: --clean option is not required anymore (should be the default). We should also remove that with this change.

**Comment by beckermarc** [comment]
@mofterdinger what do you think about this change?

**Review by mofterdinger — APPROVED** [review]
looks good to me

**Review by beckermarc — APPROVED** [review]

**Comment by beckermarc** [comment]
@MarcelMerkle Do you want to merge this PR?

**Pull request merged**